code/modules/mob/living/basic/farm_animals/pony.dm

```diff
@@ -43,7 +43,7 @@
 	AddElementTrait(TRAIT_WADDLING, INNATE_TRAIT, /datum/element/waddling)
 	AddComponent(/datum/component/tameable, food_types = list(/obj/item/food/grown/apple), tame_chance = 25, bonus_tame_chance = 15, after_tame = CALLBACK(src, PROC_REF(tamed)), unique = unique_tamer)
 
-/mob/living/basic/pony/proc/tamed(mob/living/tamer)
+/mob/living/basic/pony/tamed(mob/living/tamer, atom/food)
```

Comment:
Understand the desire but a little bit spooked by this standardisation.
I think if you are creating a contract that this _must_ be used then perhaps you should just remove the ability to pass a callback to the component and always call this proc specifically.